### PR TITLE
fix(package): make environment variables work properly for all platforms

### PR DESCRIPTION
**Pull request by BiniCodes**
Can you double check if applied right, @plumdumpling
https://www.npmjs.com/package/cross-env#cross-env-vs-cross-env-shell

Also please check if I forgot sth, I think/hope I got all of them.

### CODE REVIEW

File: examples/package.json [hunk 1]
```diff
@@ -8,21 +8,21 @@
...
-    "start:places-autocomplete-widget": "EXAMPLE_ENTRY=./places-autocomplete-widget/index.html npm run start:example",
-    "start:directions-service": "EXAMPLE_ENTRY=./directions-service/index.html npm run start:example",
-    "start:distance-matrix-service": "EXAMPLE_ENTRY=./distance-matrix-service/index.html npm run start:example",
-    "start:elevation-service": "EXAMPLE_ENTRY=./elevation-service/index.html npm run start:example",
-    "start:max-zoom-service": "EXAMPLE_ENTRY=./max-zoom-service/index.html npm run start:example",
-    "start:places-autocomplete-service": "EXAMPLE_ENTRY=./places-autocomplete-service/index.html npm run start:example",
-    "start:street-view-panorama-map": "EXAMPLE_ENTRY=./street-view-panorama-map/index.html npm run start:example",
-    "start:street-view-panorama-element": "EXAMPLE_ENTRY=./street-view-panorama-with-element/index.html npm run start:example",
+    "start:example": "npm run clean-examples && cross-env $(cat .env) cross-env PARCEL_AUTOINSTALL=false parcel serve $EXAMPLE_ENTRY --dist-dir public --port 1234 --no-cache",
```
Comment (plumdumpling) [inline examples/package.json:11]:
```suggestion
    "start:example": "npm run clean-examples && cross-env $(cat .env) PARCEL_AUTOINSTALL=false parcel serve $EXAMPLE_ENTRY --dist-dir public --port 1234 --no-cache",
```

Shouldn't be needed twice (see https://github.com/kentcdodds/cross-env/issues/15)

File: package.json [hunk 2]
```diff
@@ -10,20 +10,20 @@
...
-    "start:places-service-with-element-example": "EXAMPLE=places-service-with-element run-p start:library start:example",
-    "start:places-autocomplete-widget-example": "EXAMPLE=places-autocomplete-widget run-p start:library start:example",
-    "start:directions-service-example": "EXAMPLE=directions-service run-p start:library start:example",
-    "start:distance-matrix-service-example": "EXAMPLE=distance-matrix-service run-p start:library start:example",
-    "start:elevation-service-example": "EXAMPLE=elevation-service run-p start:library start:example",
-    "start:max-zoom-service-example": "EXAMPLE=max-zoom-service run-p start:library start:example",
-    "start:places-autocomplete-service-example": "EXAMPLE=places-autocomplete-service run-p start:library start:example",
-    "start:street-view-panorama-map-example": "EXAMPLE=street-view-panorama-map run-p start:library start:example",
-    "start:street-view-panorama-element-example": "EXAMPLE=street-view-panorama-element run-p start:library start:example",
+    "start:map-example": "cross-env EXAMPLE=map run-p start:library start:example",
```
Comment (plumdumpling) [inline package.json:13]: Please add `cross-env` to the `devDependencies` in this package.json.

**Review by plumdumpling — COMMENTED** [review]
> ### Windows Issues
> Please note that npm uses cmd by default and that doesn't support command substitution, so if you want to leverage that, then you need to update your .npmrc to set the script-shell to powershell. https://github.com/kentcdodds/cross-env/issues/192#issuecomment-513341729.

https://github.com/kentcdodds/cross-env/blob/master/README.md#windows-issues

I think we also have to adjust the .npmrc as described [here](https://github.com/kentcdodds/cross-env/issues/192#issuecomment-513341729) for Windows users.

**Review by plumdumpling — APPROVED** [review]
Let's see if it works 🙂

**Comment by BiniCodes** [comment]
> Let's see if it works 🙂

Needed to remove it and add a note for windows users as tests are failing this way. 
https://github.com/ubilabs/google-maps-react-hooks/actions/runs/3463634036/jobs/5784142887

I guess it is what he is saying here: 
> Now I have to figure out how to use a different .npmrc per platform so every developer doesn't have to create their own config file. 😕

@plumdumpling Do you think we should add the note to all the example readmes below the api key instruction as well?

File: package.json (hunk 2 above)
Comment (plumdumpling) [reply package.json:13]: ☝️

**Comment by plumdumpling** [comment]
> @plumdumpling Do you think we should add the note to all the example readmes below the api key instruction as well?

I think this is only needed in the development and to start the examples, right? So I think you can remove the Windows instructions from the library README and add it to the examples README.
Can you please also add a comment to our internal Troubleshooting issue, so that we don't forget to add this to the Troubleshooting Guide.

File: package.json (hunk 2 above)
Comment (BiniCodes) [reply package.json:13]: Oopsi

**Comment by BiniCodes** [comment]
> > @plumdumpling Do you think we should add the note to all the example readmes below the api key instruction as well?
> 
> I think this is only needed in the development and to start the examples, right? So I think you can remove the Windows instructions from the library README and add it to the examples README. Can you please also add a comment to our internal Troubleshooting issue, so that we don't forget to add this to the Troubleshooting Guide.

As far as I understood in the issue, the installation of the library neither worked - because of the env variables. But this could also be because of the `postinstall` in the older version.

**Pull request merged**